custom/icds_reports/management/commands/update_thr_fix_data.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from django.core.management.base import BaseCommand
+
+from django.db import connections, transaction
+
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+@transaction.atomic
```

Comment:
no need of this

Reply:
hey, we need the transaction to be atomic as its not a fetch query, its actually touching the data.

Reply:
cool 👍